experimental/addons/bytebase/template.cue

```diff
@@ -0,0 +1,88 @@
+package main
+
+output: {
+	apiVersion: "core.oam.dev/v1beta1"
+	kind:       "Application"
+	spec: {
+		components: [
+			{
+				name: "ns-bytebase"
+				type: "k8s-objects"
+				properties: objects: [{
+					kind:       "Namespace"
+					apiVersion: "v1"
+					metadata:
+						name: parameter.namespace
+				}]
+			},
+			{
+				name: "bytebase"
+				type: "helm"
+				properties: {
+					repoType:        "helm"
+					url:             "https://bytebase.github.io/bytebase"
+					chart:           "bytebase"
+					version:         parameter.chartVersion
+					targetNamespace: parameter.namespace
+					values: {
+						bytebase: {
+							option: {
+								port:           parameter.bytebasePort,
+								"external-url": parameter.externalURL,
+								pg:             parameter.postgresURL
+							},
+							version: parameter.version
```

Comment:
can we use the deployment yaml instead of helm chart?

Reply:
Done 👍🏽